pkg/metrics/metrics.go

```diff
@@ -31,14 +31,16 @@ const (
 	metricNamespace             = "ark"
 	backupTarballSizeBytesGauge = "backup_tarball_size_bytes"
 	// TODO: Rename the Count variables to match their strings
-	backupAttemptCount           = "backup_attempt_total"
-	backupSuccessCount           = "backup_success_total"
-	backupFailureCount           = "backup_failure_total"
-	backupDurationSeconds        = "backup_duration_seconds"
-	restoreAttemptTotal          = "restore_attempt_total"
-	restoreValidationFailedTotal = "restore_validation_failed_total"
-	restoreSuccessTotal          = "restore_success_total"
-	restoreFailedTotal           = "restore_failed_total"
+	backupAttemptCount                = "backup_attempt_total"
+	backupSuccessCount                = "backup_success_total"
+	backupFailureCount                = "backup_failure_total"
+	backupDurationSeconds             = "backup_duration_seconds"
+	restoreAttemptTotal               = "restore_attempt_total"
+	restoreValidationFailedTotal      = "restore_validation_failed_total"
+	restoreSuccessTotal               = "restore_success_total"
+	restoreFailedTotal                = "restore_failed_total"
+	volumeSnapshotsAttemptedPerBackup = "volume_snapshot_attempted_per_backup"
```

Comment:
This is probably better named `volume_snapshot_attempt_total`

Reply:
Won't that be confusing - across backups vs per backup?

Reply:
Prometheus metrics are a bit "weird" compared to what you might expect. If we do this per backup, then the individual metrics will balloon - you'd have

```
volume_snapshot_attempted_per_backup[backup-1]=12
volume_snapshot_attempted_per_backup[some-other-backup]=3
...
```

@rosskukulinski may be able to provide some better guidance here

Reply:
So the current implementation will not give us metrics per backup? Because that's what the issue description mentions. So I'm a little confused here 😅

Reply:
We generally record metrics per schedule. It's possible the issue was misleading. What I think we're really after is
- # of attempted volume snapshots, per schedule
- # of successful volume snapshots, per schedule
- # of failed volume snapshots, per schedule